website/aws.erb

```diff
@@ -1076,6 +1076,9 @@
                                 <li>
                                     <a href="/docs/providers/aws/r/ec2_transit_gateway_route_table_propagation.html">aws_ec2_transit_gateway_route_table_propagation</a>
                                 </li>
+                                <li>
```

Comment:
You'll need to add

```
                                <li>
                                    <a href="/docs/providers/aws/d/ec2_transit_gateway_peering_attachment.html">aws_ec2_transit_gateway_peering_attachment</a>
                                </li>
```

to the `<a href="#">Data Sources</a>` section above to get Travis to go green.

Reply:
@Omarimcblack @j-nix  Let me know if you don't have cycles to complete this. Thanks.